src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -542,11 +542,24 @@ private void createParentFolder() {
      * Writes a marker file to prevent unzipping more than once.
      */
     private File unpackWorkDir(Artifact artifact) throws MojoExecutionException {
+
+	// trying normal search first, all-repo search if normal failed
```

Comment:
This comment can be turned into `getLog().debug(..)`

Reply:
Sure, done.

Btw - what ide are you using, Eclipse? If so could please you un-gitignore (or copy to some other folder) the file with formatting settings? It would help keeping the format consistent. It can also be used with https://github.com/revelc/formatter-maven-plugin so the formatting is part of the build process no matter the users' ide.

I've sent you an email, nothing urgent but please check if it didn't get stuck in spam folder

Reply:
I'm using IDEA, if you want you can add [.editorconfig](https://editorconfig.org/) which is a cross-ide formatter.

I noticed your e-mail and will respond :)

Reply:
And I prefer spaces over tabs